modules/gitops-openshift-go-terminology.adoc

```diff
@@ -0,0 +1,47 @@
+//
+// * understanding_openshift_gitops/about-redhat-openshift-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-openshift-go-terminology_{context}"]
+= OpenShift {gitops-shortname} terminology
+
+This section summarizes the basic terms used in OpenShift {gitops-shortname}:
+
+[horizontal]
+Argo CD:: An open-source declarative tool that automates the continuous deployment (CD) of Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+{gitops-shortname}:: {gitops-shortname} is a declarative way to implement continuous deployment for cloud native applications.
+{gitops-title}:: An Operator that uses Argo CD as the declarative {gitops-shortname} engine for enabling {gitops-shortname} workflows across multicluster OpenShift and Kubernetes infrastructure.
+{gitops-shortname} CLI ({gitops-shortname} `argocd` CLI):: A tool to configure and manage {gitops-title} and Argo CD resources from the command line. 
+Control plane:: A {gitops-shortname} control plane is a namespace where Argo CD is installed. A control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. Examples include a control plane repository wherein you can create one or more application custom resource and express an intent to deploy and sync Kubernetes manifests from Git.
+Workload:: Any process running within a container. Examples include a Spring Boot application, a NodeJS Express application, or a Ruby on Rails application.
+Application (Argo CD application):: A group of Kubernetes resources as defined by a manifest. This is a Custom Resource Definition (CRD). The `Application` CRD is a resource object representing a deployed application instance in an environment.
+Declarative setup:: A declarative description of the infrastructure you need in your specified environment, for system and application setup or configuration. You can specify this description in a YAML configuration file in the Git repository. The declarative setup contains an automated process to make your environment and infrastucture match the described state. Examples include defining Argo CD applications, projects and settings declaratively using YAML manifests.
+Live state:: The live state of an application. Examples include what pods and other application resources are deployed.
+Target state:: The desired state of an application, as represented by files in a Git repository.
+Sync:: The process of making an application move to its target state. Examples include syncing an application by applying changes to a cluster.
+Sync status:: The status of an application that indicates whether the live state matches the target state.
+Refresh:: The process to compare the latest code in Git with the live state and figure out the difference.
+AppProject:: A resource object and CRD representing a logical grouping of applications within a project, which governs where and how an application might manage resources. 
```

Comment:
> AppProject:: 

A resource object and CRD representing a logical grouping of applications within a project, which governs where and how an application might manage resources, **_as well as what Argo CD users may access those applications_**.

Reply:
@jgwest: A resource object and CRD representing a logical grouping of applications within a project, which governs where and how an application might manage resources. In addition, as an administrator, you use AppProject to restrict where and how non-admin Argo CD users might access those applications.